javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,303 @@
+"use strict";
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj) {
+  // Solution code here...
+  const liArray = [];
+  for (const [key, value] of Object.entries(obj)) {
+    liArray.push(`<li>${key}: ${value}</li>`);
+  }
+  return liArray;
+}
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  // Solution code here...
+  // let count = 0;
+  // input.map((int1) => {
+  //   int1.map((int2) => {
+  //     if (target === int2) {
+  //       count++;
+  //     }
+  //   });
+  // });
+  // console.log(count);
+  // return count;
```

Comment:
Nice work on these!

Feedback here is to remove this unused code to keep things clean 👍